tools-v2/pkg/cli/command/curvebs/query/scanstatus/scanstatus.go

```diff
@@ -0,0 +1,156 @@
+)
+
+type ScanStatusCommand struct {
+	CopysetInfoList []*common.CopysetInfo
+	basecmd.FinalCurveCmd
+}
+
+var _ basecmd.FinalCurveCmdFunc = (*ScanStatusCommand)(nil) // check interface
+
+func NewQueryScanStatusCommand() *ScanStatusCommand {
+	scanStatusCmd := &ScanStatusCommand{
+		FinalCurveCmd: basecmd.FinalCurveCmd{
+			Use:     "scan-status",
+			Short:   "quey ScanStatus Info in bs",
+			Example: scanStatusExample,
+		},
+	}
+
+	basecmd.NewFinalCurveCli(&scanStatusCmd.FinalCurveCmd, scanStatusCmd)
+	return scanStatusCmd
+}
+
+func NewScanStatusCommand() *cobra.Command {
+	return NewQueryScanStatusCommand().Cmd
+}
+
+func (cCmd *ScanStatusCommand) AddFlags() {
+	config.AddBsMdsFlagOption(cCmd.Cmd)
+	config.AddRpcRetryTimesFlag(cCmd.Cmd)
+	config.AddRpcTimeoutFlag(cCmd.Cmd)
+	config.AddBsLogicalPoolIdSliceRequiredFlag(cCmd.Cmd)
+	config.AddBsCopysetIdSliceRequiredFlag(cCmd.Cmd)
+}
+
+func (cCmd *ScanStatusCommand) Init(cmd *cobra.Command, args []string) error {
+	mdsAddrs := config.GetBsFlagString(cmd, config.CURVEBS_MDSADDR)
+	retrytime := config.GetBsFlagInt32(cmd, config.RPCRETRYTIMES)
+	timeout := config.GetFlagDuration(cCmd.Cmd, config.RPCTIMEOUT)
+	logicalpoolidList := config.GetBsFlagString(cCmd.Cmd, config.CURVEBS_LOGIC_POOL_ID)
+	copysetidList := config.GetBsFlagString(cCmd.Cmd, config.CURVEBS_COPYSET_ID)
+
+	cCmd.Cmd.ParseFlags([]string{
+		fmt.Sprintf("--%s", config.CURVEBS_MDSADDR), mdsAddrs,
+		fmt.Sprintf("--%s", config.RPCRETRYTIMES), fmt.Sprintf("%d", retrytime),
+		fmt.Sprintf("--%s", config.RPCTIMEOUT), fmt.Sprintf("%d", timeout),
+		fmt.Sprintf("--%s", config.CURVEBS_LOGIC_POOL_ID), logicalpoolidList,
+		fmt.Sprintf("--%s", config.CURVEBS_COPYSET_ID), copysetidList,
+	})
+
+	copysetInfoList, err := copyset.GetCopyset(cmd)
+	if err.TypeCode() != cmderror.CODE_SUCCESS {
+		return err.ToError()
+	}
+	cCmd.CopysetInfoList = copysetInfoList
+	log.Printf("copysetinfolist = %v", copysetInfoList)
+	header := []string{
+		cobrautil.ROW_LOGICALPOOL,
+		cobrautil.ROW_COPYSET_ID,
+		cobrautil.ROW_SCAN,
+		cobrautil.ROW_LASTSCANSEC,
+		cobrautil.ROW_LASTSCANCONSISTENT,
+	}
+	cCmd.SetHeader(header)
+	cCmd.TableNew.SetAutoMergeCellsByColumnIndex(cobrautil.GetIndexSlice(
+		cCmd.Header, []string{cobrautil.ROW_COPYSET_ID, cobrautil.ROW_POOL_ID,
+			cobrautil.ROW_SCAN,
+		},
+	))
+
+	return nil
+}
+
+func (cCmd *ScanStatusCommand) Print(cmd *cobra.Command, args []string) error {
+	return output.FinalCmdOutput(&cCmd.FinalCurveCmd, cCmd)
+}
+
+func (cCmd *ScanStatusCommand) RunCommand(cmd *cobra.Command, args []string) error {
+	rows := make([]map[string]string, 0)
+	for _, info := range cCmd.CopysetInfoList {
+		row := make(map[string]string)
+		row[cobrautil.ROW_LOGICALPOOL] = fmt.Sprintf("%d", info.GetLogicalPoolId())
+		row[cobrautil.ROW_COPYSET_ID] = fmt.Sprintf("%d", info.GetCopysetId())
+		row[cobrautil.ROW_SCAN] = fmt.Sprintf("%v", info.GetScaning())
+		row[cobrautil.ROW_LASTSCANSEC] = fmt.Sprintf("%v", info.GetLastScanSec())
```

Comment:
This should be a Unix Timestamp, you can convert it.

Reply:
I don't find convert in the old tools.  seems the old tools just out the lastscansec?

Reply:
> I don't find convert in the old tools. seems the old tools just out the lastscansec?

There is no need to emulate old tools, you can implement it in [time.go](https://github.com/opencurve/curve/blob/master/tools-v2/internal/utils/time.go).

Reply:
> > I don't find convert in the old tools. seems the old tools just out the lastscansec?
> 
> There is no need to emulate old tools, you can implement it in [time.go](https://github.com/opencurve/curve/blob/master/tools-v2/internal/utils/time.go).

done